test/data.jl

```diff
@@ -73,6 +73,18 @@ module TestData
 
     #test_group("ref")
     @test sdf6a[1,2] == 4
```

Comment:
Doesn't look like the new tests are related to this line: add a new "getindex" section.